package.json

```diff
@@ -30,7 +30,7 @@
     "js-beautify": "^1.11.0",
     "prettier": "^1.18.2",
     "prettydiff2": "^2.2.8",
-    "tree-sitter-cli": "^0.16.8",
+    "tree-sitter-cli": "^0.16.9",
```

Comment:
**Line 4-6: the `details` object**

Can you update both attributes within this object to reflect the current synced version of `tree-sitter/tree-sitter-python`? This helps us stay up to date with new commits to their repository. 👍  So for instance, `"latest-commit": "58f57240834d6b88624e32ad0ab9531d55fb7a5d"`. 

(This isn't for line 33, but I can't review the line that needs updating.)